File: android-app/app/src/main/java/com/tecazuay/example/android_app/models/Usuario.java
package com.tecazuay.example.android_app.models;

public class Usuario {

    private Long personaId;
    private String nombres;
    private String apellidos;
    private String correo;
    private String password;
    private String token;
    private String telefono;
    private Rol rol;

    public Usuario() {
    }

    public Usuario(Long personaId, String nombres, String apellidos, String correo, String password, String token, String telefono, Rol rol) {
        this.personaId = personaId;
        this.nombres = nombres;
        this.apellidos = apellidos;
        this.correo = correo;
        this.password = password;
        this.token = token;
        this.telefono = telefono;
        this.rol = rol;
    }

    public Long getPersonaId() {
        return personaId;
    }

    public void setPersonaId(Long personaId) {
        this.personaId = personaId;
    }

    public String getNombres() {
        return nombres;
    }

    public void setNombres(String nombres) {
        this.nombres = nombres;
    }

    public String getApellidos() {
        return apellidos;
    }

    public void setApellidos(String apellidos) {
        this.apellidos = apellidos;
    }

    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }

    public String getTelefono() {
        return telefono;
    }

    public void setTelefono(String telefono) {
        this.telefono = telefono;
    }

    public Rol getRol() {
        return rol;
    }

    public void setRol(Rol rol) {
        this.rol = rol;
    }
}

File: angular-web/src/app/models/usuario.ts
import { Rol } from './rol';

export class Usuario {
  personaId: number;
  cedula: string
  nombres: string;
  apellidos: string;
  correo: string;
  telefono: string;
  password: string;
  token: string;
  nombreCompleto?: string;
  rol: Rol;
}

export interface LoginForm {
  message?: string;
  errors?: LoginFormErrors;
}

interface LoginFormErrors {
  correo: string[];
  password: string[];
}

export interface LoginUser {
  apellidos: string;
  nombres: string;
  correo: string;
  rol: string;
  token: string;
  type: number;
}

export interface ComboUsuario {
  usuario_id: string;
  nombre_completo: string;
}
